apps/Epiphany Web Browser/description

```diff
@@ -0,0 +1,5 @@
+Epiphany is a GNOME web browser based on the WebKit rendering engine. Also known as GNOME Web.
+
+Web is the web browser for the GNOME desktop and for elementary OS, based on the popular WebKit engine. It offers a simple, clean, beautiful view of the web featuring first-class GNOME and Pantheon desktop integration, a built-in adblocker enabled by default, and Intelligent Tracking Prevention. With no useless widgets or wasted space, Web closely follows GNOME's design philosophy. 
```

Comment:
This line reads like an advertisement.

Reply:
So what to do?

Reply:
> So what to do?

Try to make it sound less opinionated. This line contains many buzz-words like `popular`, `simple`, clean`, `beautiful`, `first-class`, `intelligent`, `without useless widgets`, etc.
Pi-Apps has descriptions to help users make an informed choice, not to make a sales pitch.

If I were writing the description, I'd mention that Epiphany used to be Raspbian's default browser back in the day, before it began showing its age and they had to switch to Chromium.

Reply:
What about https://github.com/Botspot/pi-apps/pull/1432/commits/9674c2bd30406b3086877088b4b45cf10f40ba09